app/scripts/modules/amazon/src/serverGroup/configure/AmazonImageSelectInput.tsx

```diff
@@ -343,7 +343,10 @@ export class AmazonImageSelectInput extends React.Component<IAmazonImageSelector
             filterOptions={false as any}
             noResultsText={searchNoResultsText}
             options={searchResults}
-            onInputChange={searchInput => this.searchInput$.next(searchInput)}
+            onInputChange={searchInput => {
+              this.searchInput$.next(searchInput);
+              return searchInput;
```

Comment:
This was a side effect of the update. This function should return a string, while enabling `serchInput$.next(...)` to continue being invoked properly.